azure-pipelines.yml

```diff
@@ -205,6 +205,10 @@ jobs:
       TargetPath: '$(Build.ArtifactStagingDirectory)/metadata'
       artifactName: metadata
 
+  - task: PipAuthenticate@1
+    displayName: 'Pip Authenticate'
+    inputs:
+      artifactFeeds: 'public/azure-cli-feed'
```

Comment:
This task set `PIP_INDEX_URL` so that this feed can be used later.

Ref: https://learn.microsoft.com/en-us/azure/devops/pipelines/tasks/reference/pip-authenticate-v1?view=azure-pipelines